package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "author": "jitsi.org",
```

Comment:
"Fellow Jitsters" ? and set the email to support@jitsi.org ?

Reply:
I've added the email (and updated a few other fields, but I don't think we should use the "Fellow Jitsters" as author